pkg/es/client.go

```diff
@@ -27,6 +27,7 @@ type Client interface {
 	Index() IndexService
 	Search(indices ...string) SearchService
 	MultiSearch() MultiSearchService
+	Close() error
```

Comment:
What about embedding io.Closer here?

I would like to close this in in main case of sigterm, however there is present only spanwriter. Do we want to add close to spanwriter too?

Reply:
Lately I prefer not adding Close to interfaces. The main often creates services with explicit type, i.e. `*SomeStruct`, in which case it already has access to Close(). In cases when we use an abstract factory pattern and get an interface back, we can still call close via conditional cast:
```go
if closer, ok := someComponent.(io.Closer); ok {
    closer.Close()
}
```

Reply:
I had to add `close` here to be able close the bulk service it in es span writer.